recipes/tiny-dnn/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.0a3":
```

Comment:
This project is abandoned.
https://github.com/tiny-dnn/tiny-dnn#the-project-may-be-abandoned-since-the-maintainers-are-just-looking-to-move-on-in-the-case-anyone-is-interested-in-continuing-the-project-let-us-know-so-that-we-can-discuss-next-steps
Let's add the latest commit as cci version as well?

Reply:
that's a good option!

Reply:
Done! latest version available.